apps/cli/src/web/deploy.ts

```diff
@@ -15,71 +15,31 @@ import type {
   SiteUploadResponse,
 } from '@curvenote/blocks';
 import type { ISession } from '../session/types';
-import { getLogoPaths } from '../site/manifest';
 
 type FromTo = {
   from: string;
   to: string;
 };
 
-function listConfig(session: ISession): FromTo[] {
-  const siteConfig = selectors.selectCurrentSiteConfig(session.store.getState());
-  if (!siteConfig) throw new Error('🧐 No site config found.');
-  const paths: FromTo[] = [];
-  paths.push({
-    from: path.join(session.serverPath(), 'app', 'config.json'),
-    to: 'config.json',
-  });
-  // Ensure that we can access the logo
-  const logo = getLogoPaths(session, siteConfig.logo, { silent: true });
-  if (logo) {
-    paths.push({ from: logo.public, to: `public${logo.url}` });
-  }
-  if (siteConfig.favicon) {
-    const favicon = path.basename(siteConfig.favicon);
-    paths.push({
-      from: path.join(session.serverPath(), 'public', favicon),
-      to: `public/${favicon}`,
-    });
-  }
-  // Load all static action resources
-  siteConfig.actions?.forEach((action) => {
-    if (!action.static) return;
-    // String leading slash
-    const names = action.url.split('/').filter((s) => s);
-    paths.push({
-      from: path.join(session.serverPath(), 'public', ...names),
-      to: `public/${names.join('/')}`,
-    });
-  });
-  return paths;
-}
-
-function listContentFolders(session: ISession): FromTo[] {
-  const contentFolder = path.join(session.serverPath(), 'app', 'content');
-  const folders = fs.readdirSync(contentFolder);
-  const fromTo = folders.map((folderName) => {
-    const basePath = path.join(contentFolder, folderName);
-    const files = fs.readdirSync(basePath);
-    return files.map((f) => ({
-      from: path.join(basePath, f),
-      to: `content/${folderName}/${f}`,
-    }));
-  });
-  return fromTo.flat();
-}
-
-function listPublic(session: ISession): FromTo[] {
-  const staticFolder = path.join(session.publicPath(), '_static');
-  if (!fs.existsSync(staticFolder)) return [];
-  const assets = fs.readdirSync(staticFolder);
-  const fromTo = assets.map((assetName) => {
-    return {
-      from: path.join(staticFolder, assetName),
-      to: `public/_static/${assetName}`,
-    };
+function listFolderContents(session: ISession, from: string, to = ''): FromTo[] {
```

Comment:
Nope, it was super complex before because it was all scattered. Now what is in site is also on the CDN.

This will also make it easier to copy in other content into this folder as well.